src/components/pages/docs.js

```diff
@@ -129,13 +136,17 @@ class Docs extends React.Component {
 
   handleCloneClick(doc) {
     const { t } = this.props;
-    this.setState({
-      newDocTitle: doc.title ? `${doc.title} ${t('copyTitleSuffix')}` : t('defaultDocumentTitle'),
-      newDocLanguage: doc.language,
-      newDocSlug: doc.slug ? `${doc.slug}-${t('copySlugSuffix')}` : DEFAULT_DOCUMENT_SLUG,
-      newDocBlueprintKey: doc.key,
+    this.setState(prevState => ({
+      newDoc: {
+        ...prevState.newDoc,
+        title: doc.title ? `${doc.title} ${t('copyTitleSuffix')}` : t('defaultDocumentTitle'),
+        language: doc.language,
+        slug: doc.slug ? `${doc.slug}-${t('copySlugSuffix')}` : DEFAULT_DOCUMENT_SLUG,
+        tags: doc.tags || [],
```

Comment:
Would be better to spread doc.tags here and create a new array in order to avoid any nasty side effects.

Reply:
Same here

Reply:
Ahm, I might have said something dumb though, spreading will not work if the doc.tags are not yet defined, so we need : doc.tags?.slice(0) || []
Thanks for changing, I had my mind that we'd forget when introducing the tags editor on this thing.

Reply:
I already covered the undefined case, no worries, didn't take the comment ad litteram ☺️